GenerateCLP/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 #-----------------------------------------------------------------------------
-cmake_minimum_required(VERSION 3.5)
```

Comment:
I suggest we remove support for building GenerateCLP as a standalone project in a different set of commit

Reply:
Done.